app/views/simplified-account/settings/service-name/index.njk

```diff
@@ -27,7 +27,7 @@
             href: manageEn,
             text: "Change",
             classes: "govuk-link--no-visited-state",
-            visuallyHiddenText: "Edit English service name",
+            visuallyHiddenText: "English service name",
```

Comment:
Why is the "Edit" removed? I would've thought a verb would be useful for accessible content.

Reply:
I put a bullet point about this in the PR description, but it's a bit buried.  "I noticed a common issue with the use of visually hidden text in Summary Lists and corrected it everywhere I could see it".

Basically the old version outputs:

```
Edit <span class="govuk-visually-hidden">Edit English service name</span>
```

Which would read as "Edit Edit English service name", it's not clear from the variable names but you can see it in the examples on the Design System https://design-system.service.gov.uk/components/summary-list/ (you can compare the Nunjucks input to the HTML output).